server/app/services/triplers.js

```diff
@@ -597,12 +592,15 @@ async function setTriplerEkataAssociatedPeople(tripler, verification) {
   match (t:Tripler {id:$t_id})
   merge (e:EkataPerson {id:$e_id})
   merge (t)-[r:EKATA_ASSOCIATED]->(e)
+  SET e += {first_name=$e_first, e.last_name=$e_last}
```

Comment:
Do we need to do any uppercasing on first letter?

Reply:
i just wanted to put whever ekata sent so we know what's what